src/plone/restapi/tests/test_statictime.py

```diff
@@ -25,6 +29,12 @@
 import transaction
 import unittest
 
+padiscussion_version = get_distribution("plone.app.discussion").version
+if parse_version(padiscussion_version) >= parse_version("4.0.0b4.dev1"):
+    NEW_PADISCUSSION = True
+else:
+    NEW_PADISCUSSION = False
```

Comment:
@instification In general, it's better to test for the presence of some feature rather than checking versions. I also have some suspicions about the effects of using get_distribution on import performance. What about something like this?
```
import plone.app.discussion.browser.comment
NEW_PADISCUSSION = hasattr(plone.app.discussion.browser.comment, "localized_now")
```

(Maybe even call the variable something more meaningful like HAS_TZ_AWARE_COMMENTS)